src/auth-service/config/environments/development.js

```
@@ -26,6 +28,7 @@ const devConfig = {
   KAFKA_RAW_MEASUREMENTS_TOPICS: process.env.KAFKA_RAW_MEASUREMENTS_TOPICS_DEV,
   KAFKA_CLIENT_ID: process.env.KAFKA_CLIENT_ID_DEV,
   KAFKA_CLIENT_GROUP: process.env.KAFKA_CLIENT_GROUP_DEV,
+  //redis
   REDIS_SERVER: process.env.DEV_REDIS_SERVER,
   REDIS_PORT: process.env.DEV_REDIS_PORT,
```

Biome (1.9.4)

[error] This property value named REDIS_PORT is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named REDIS_PORT


(lint/suspicious/noDuplicateObjectKeys)